sound/soc/intel/boards/sdw_rt711_i2s_rt1308.c

```diff
@@ -192,6 +205,49 @@ static struct snd_soc_ops rt1308_ops = {
 	.hw_params = rt1308_hw_params,
 };
 
+static int sof_rt711_rt1308_quirk_cb(const struct dmi_system_id *id)
+{
+	sof_rt711_rt1308_quirk = (unsigned long)id->driver_data;
+	return 1;
+}
+
+static const struct dmi_system_id sof_sdw_rt711_rt1308_quirk_table[] = {
+	{
+		.callback = sof_rt711_rt1308_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "IntelCorporation"),
```

Comment:
spaces?
and board name?

existing examples are:

.matches = {
			DMI_MATCH(DMI_SYS_VENDOR, "Intel Corporation"),
			DMI_MATCH(DMI_PRODUCT_NAME, "WhiskeyLake Client"),
		},

Reply:
Thanks, I checked there is no space.   I will add product name